components/ReviewSection.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  Heading,
+  HeadingLevels,
+  Paragraph,
+  SummaryList,
+} from "lbh-frontend-react";
+import React from "react";
+import PropTypes from "../helpers/PropTypes";
+import { SectionRow } from "../helpers/useReviewSectionRows";
+
+interface Props {
+  section?: {
+    heading?: string;
+    rows?: SectionRow[];
+  };
+}
+
+export const ReviewSection: React.FunctionComponent<Props> = (props) => {
+  const { section } = props;
+  const { heading, rows } = section || {};
+
+  return (
+    <React.Fragment key={heading}>
```

Comment:
```suggestion
    <>
```

`key` doesn't mean anything on the components themselves, I don't think. (It does, but it's not relevant in this case.)